### PR TITLE
feat: openapi schema

### LINKED ISSUE #291: Create Open API schema for HTTP API

### PR DESCRIPTION
**Pull request by alanshaw**
resolves #291

### CODE REVIEW

File: packages/website/public/schema.yml [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+                $ref: '#/components/schemas/UploadResponse'
+        '400':
+          $ref: '#/components/responses/badRequest' 
+        '401':
+          $ref: '#/components/responses/unauthorized'
+        '403':
+          $ref: '#/components/responses/forbidden'
+        '5XX':
+          $ref: '#/components/responses/internalServerError'
+  /:
```
Comment (olizilla) [inline packages/website/public/schema.yml:83]:
```suggestion
  /user/uploads:
```

File: packages/website/public/schema.yml [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ListResponse'
+        '401':
+          $ref: '#/components/responses/unauthorized'
+        '403':
+          $ref: '#/components/responses/forbidden'
+        '5XX':
+          $ref: '#/components/responses/internalServerError'
+  /{cid}:
```
Comment (olizilla) [inline packages/website/public/schema.yml:105]:
```suggestion
   /user/uploads/{cid}:
```

**Comment by olizilla** [comment]
oh this is a draft. i stop.

**Comment by alanshaw** [comment]
@johndmulhausen is this ready for review?

**Comment by olizilla** [comment]
the openapi schema now lives at https://docs.web3.storage/schema.yml and the source is https://github.com/web3-storage/docs/blob/main/docs/.vuepress/public/schema.yml

**Pull request closed without merging**